### PR TITLE
[WIP] Add Kubernetes User Formation

### PR DESCRIPTION
**Pull request by ameukam**
Improve the current course related the kubernetes' user formation.

### CODE REVIEW

File: cours.json [hunk 1]
```diff
@@ -95,6 +95,22 @@
...
     ]
   },
+  "kubernetes-user": {
+    "course_name": "kubernetes-user",
+    "course_description": "Formation Kubernetes Utilisateur",
+    "modules": [
+      "intro-osones",
+      "containers/kubernetes/intro-osones",
+      "containers/kubernetes/k8s-governance",
+      "containers/kubernetes/k8s-architecture",
```
Comment (Adri2000) [inline cours.json]: is that really for a *user* training?

File: cours/containers/k8s-architecture.fr.md [hunk 2]
```diff
@@ -27,7 +27,7 @@
 - Service principal de Kubernetes
 
 - Permet à Kubernetes de s'auto configurer :
-    - Surveille un dossier contenant les *manifests* (fichiers YAML des différents composant de K8s).
+    - Surveille un dossier contenant les *manifests* (fichiers YAML des différents composant de Kubnertes).
```
Comment (Adri2000) [inline cours/containers/k8s-architecture.fr.md]: typo

File: cours/containers/kubernetes/k8s-architecture.fr.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
+# Kubernetes : Architecture {-}
+
+### Kubernetes : Composants
+
+- Kubernetes est écrit en Go, compilé statiquement.
+
```
Comment (Adri2000) [inline cours/containers/kubernetes/k8s-architecture.fr.md]: I'd not put blank lines between different points in a single slide, just for readability of the md file

File: cours/containers/kubernetes/k8s-architecture.fr.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
+# Kubernetes : Architecture {-}
```
Comment (Adri2000) [inline cours/containers/kubernetes/k8s-architecture.fr.md]: why {-} here?

File: cours/containers/docker-orchestration.fr.md [hunk 5]
```diff
@@ -124,7 +124,7 @@ docker
 
 ![](images/docker/mesos.png){height="100px"}
```
Comment (Adri2000) [inline cours/containers/docker-orchestration.fr.md]: image description would be nice (inside [])

File: cours/containers/kubernetes/k8s-architecture.fr.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+- De base dans Kubernetes : replication controller, endpoints controller, namespace controller et serviceaccounts controller
+
+### Kubernetes : network-policy-controller
+
+- Implémente l'objet NetworkPolicy
+
+- Contrôle la communication entre les Pods
+
+- Externe à Kubernetes et implémenté par la solution de Networking choisie :
+    - [Calico](https://projectcalico.org/)
```
Comment (Adri2000) [inline cours/containers/kubernetes/k8s-architecture.fr.md]: that kind of links won't be visible when printed on dead tree

File: cours/containers/docker-orchestration.fr.md (hunk 5 above)
Comment (Adri2000) [reply cours/containers/docker-orchestration.fr.md]: sorry that's actually not part of your patch :) but same issue in other parts of your patch

File: cours.json (hunk 1 above)
Comment (rguichard) [reply cours.json]: We'll all discuss about the list of modules inside the kubernetes user/ops training

File: cours/containers/kubernetes/k8s-architecture.fr.md (hunk 4 above)
Comment (ameukam) [reply cours/containers/kubernetes/k8s-architecture.fr.md]: i wanted to simulate a behavior. forgot to remove it. thanks.

**Pull request closed without merging**